PS D:\PROG\zyablov_netology\docker-test> docker pull busybox
Using default tag: latest
latest: Pulling from library/busybox
3f4d90098f5b: Pull complete
Digest: sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79
Status: Downloaded newer image for busybox:latest
docker.io/library/busybox:latest
What's Next?
  1. Sign in to your Docker account → docker login

PS D:\PROG\zyablov_netology\docker-test> docker run --name pinger -it busybox ping -c 7 netology.ru
PING netology.ru (188.114.98.234): 56 data bytes
64 bytes from 188.114.98.234: seq=0 ttl=63 time=6.094 ms
64 bytes from 188.114.98.234: seq=1 ttl=63 time=6.483 ms
64 bytes from 188.114.98.234: seq=2 ttl=63 time=6.179 ms
64 bytes from 188.114.98.234: seq=3 ttl=63 time=4.710 ms
64 bytes from 188.114.98.234: seq=4 ttl=63 time=4.499 ms
64 bytes from 188.114.98.234: seq=5 ttl=63 time=6.321 ms
64 bytes from 188.114.98.234: seq=6 ttl=63 time=4.396 ms

--- netology.ru ping statistics ---
7 packets transmitted, 7 packets received, 0% packet loss

PS D:\PROG\zyablov_netology\docker-test> docker ps -a
CONTAINER ID   IMAGE     COMMAND                  CREATED          STATUS                      PORTS     NAMES
cc6eefae10bf   busybox   "ping -c 7 netology.…"   25 seconds ago   Exited (0) 18 seconds ago             pinger
PING netology.ru (188.114.98.234): 56 data bytes
64 bytes from 188.114.98.234: seq=0 ttl=63 time=6.094 ms
64 bytes from 188.114.98.234: seq=1 ttl=63 time=6.483 ms
64 bytes from 188.114.98.234: seq=2 ttl=63 time=6.179 ms
64 bytes from 188.114.98.234: seq=3 ttl=63 time=4.710 ms
64 bytes from 188.114.98.234: seq=4 ttl=63 time=4.499 ms
64 bytes from 188.114.98.234: seq=5 ttl=63 time=6.321 ms
64 bytes from 188.114.98.234: seq=6 ttl=63 time=4.396 ms

--- netology.ru ping statistics ---
7 packets transmitted, 7 packets received, 0% packet loss
round-trip min/avg/max = 4.396/5.526/6.483 ms

PS D:\PROG\zyablov_netology\docker-test> docker run --name pinger -it busybox ping -c 7 netology.ru
docker: Error response from daemon: Conflict. The container name "/pinger" is already in use by container "cc6eefae10bf11a3e41a66107efa61d306250e6c2645a495f0bffe57b452e64f". You have to remove (or rename) that container to be able to reuse that name.

PS D:\PROG\zyablov_netology\docker-test> docker logs pinger
64 bytes from 188.114.98.234: seq=0 ttl=63 time=6.094 ms
64 bytes from 188.114.98.234: seq=1 ttl=63 time=6.483 ms
64 bytes from 188.114.98.234: seq=2 ttl=63 time=6.179 ms
64 bytes from 188.114.98.234: seq=3 ttl=63 time=4.710 ms
64 bytes from 188.114.98.234: seq=4 ttl=63 time=4.499 ms
64 bytes from 188.114.98.234: seq=5 ttl=63 time=6.321 ms
64 bytes from 188.114.98.234: seq=6 ttl=63 time=4.396 ms

--- netology.ru ping statistics ---
7 packets transmitted, 7 packets received, 0% packet loss
round-trip min/avg/max = 4.396/5.526/6.483 ms

PS D:\PROG\zyablov_netology\docker-test> docker ps -a
CONTAINER ID   IMAGE     COMMAND                  CREATED          STATUS                      PORTS     NAMES
cc6eefae10bf   busybox   "ping -c 7 netology.…"   58 seconds ago   Exited (0) 51 seconds ago             pinger

PS D:\PROG\zyablov_netology\docker-test> docker rm -f pinger
pinger

PS D:\PROG\zyablov_netology\docker-test> docker rmi busybox
Untagged: busybox:latest
Untagged: busybox@sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79
Deleted: sha256:a416a98b71e224a31ee99cff8e16063554498227d2b696152a9c3e0aa65e5824
Deleted: sha256:3d24ee258efc3bfe4066a1a9fb83febf6dc0b1548dfe896161533668281c9f4f

PS D:\PROG\zyablov_netology\docker-test> 